src/Codec/Xlsx/Types/Drawing/Common.hs

```diff
@@ -462,7 +462,7 @@ colorChoiceFromNode n
     return $ RgbColor val
   | otherwise = fail "no matching color choice node"
 
-coordinate :: Monad m => Text -> m Coordinate
+coordinate :: Text -> [Coordinate]
```

Comment:
Is there some motivation to switch to list and not use `MonadFail`?

Reply:
it needed changing either way, MonadFail wasn't in 7 and it was only used as list

Reply:
Actually I think the source of this problem is a bad naming I used: something like `parseCoordinate` would be much better and in that sense `MonadFail` would make even more sense I think. `MonadFail` for lists is quite unintuitive by itself but without `MonadFail` encoding it just as list in output type makes the sense of this function even more obscure.
What do you think about `parseXXX` + `MonadFail` instead of functions returning list as a variant of `Maybe`?
This function is just a helper for sure but I think it makes sense to make the API cleaner if we can.
Does this make sense or maybe I miss something here and this needs to be resolved some other way around?

Reply:
I didn't actually notice it was public 🤦‍♂️ for the sake of anyone updating I would be tempted to leave it despite the possibility better name.

Reply:
Ok, let's keep the name then